src/classes/tmConstants.cls

```diff
@@ -0,0 +1,95 @@
+public virtual with sharing class tmConstants {
+    public static String CSP_TENANT_TYPE_FORCEDOTCOM { get {return 'force.com';} private set;}
+    public static Set<String> VALID_EDITIONS {
+        get {
+            if (VALID_EDITIONS == null)
+                VALID_EDITIONS = new Set<String> {'unlimited','enterprise','professional','group','performance'};
+
+            return VALID_EDITIONS;
+        }
+        private set;
```

Comment:
Variables should use the `lowerCamelCase` naming convention. Please rename accordingly.